scripts/sanity_chk/sanitylib.py

```diff
@@ -1208,12 +1208,6 @@ def get_test(self, name, valid_keys):
             d[k] = v
 
         for k, v in self.tests[name].items():
-            if k not in valid_keys:
-                raise ConfigurationError(
-                    self.filename,
-                    "Unknown config key '%s' in definition for '%s'" %
-                    (k, name))
-
```

Comment:
What is the reason behind this change? Is it because 'integration' is not in the list of valid_keys? Why not adding 'integration' there and keeping this check in place?

Reply:
this is already validated by the yaml schema checks, it is just an old check we had that is duplicating what yaml verification does now.